src/app/views/marketplace/marketplace-app.html

```diff
@@ -91,26 +91,134 @@ <h2 class="media-heading">{{vm.app.name}}</h2>
           <!-- End pics/details row -->
         </div>
 
+        <!-- Description row -->
+        <div class="row">
+          <div class="col-md-12">
+            <div class="well top-buffer-1" ng-bind-html="vm.app.description"></div>
+          </div>
+        </div>
+
         <!-- Testimonials row -->
         <div class="row">
-          <div class="col-md-6" ng-repeat="testimonial in vm.app.testimonials | limitTo:2" ng-show="vm.isTestimonialShown(testimonial)">
```

Comment:
Do not remove testimonials as they exists nowadays. Reviews/Questions are different than testimonials, which are commercial content.